src/table/utils/styling-utils.js

```diff
@@ -96,26 +118,26 @@ export function getColumnStyle(styling, qAttrExps, stylingInfo) {
   };
 }
 
-export function getSelectionColors(background = STYLING_DEFAULTS.WHITE, cell, selectionState) {
+export function getSelectionColors(cell, selectionState, background, tableBackgroundColor) {
   const { colIdx, rows, api } = selectionState;
 
   if (api.isModal()) {
-    if (colIdx !== cell.colIdx) {
-      return { background: `${STYLING_DEFAULTS.EXCLUDED_BACKGROUND}, ${background}` };
-    }
+    if (colIdx !== cell.colIdx)
+      return {
+        background: `${STYLING_DEFAULTS.EXCLUDED_BACKGROUND}, ${
+          background || tableBackgroundColor || STYLING_DEFAULTS.WHITE
```

Comment:
Can both `background` and `tableBackgroundColor` be undefined? also, which one is from the theme?

Reply:
isn't better to have a default value for the parameter?

Reply:
Yes, they can.
background is from qAttributeExpressions in qDimensions or qMeasures
tableBackgroundColor is from the theme
I am adding the comments to clarify.
Yes, it is better `tableBackgroundColor = STYLING_DEFAULTS.WHITE`

Reply:
maybe call them `columnBackground` and `themeBackground` so you know which one is which? I know I'm being picky about variable names but I think we need to be consistent when we add this extra layer of theming to the styling.

Reply:
happy to hear that. I am doing a similar refactor.

Reply:
But I realized that renaming should be in a new PR otherwise conflicts everywhere. Sorry for so many PRs.